docs/_includes/footer.html

```diff
@@ -56,23 +56,23 @@
             l24-7.8c10.5-3.6,16.2-15,12.6-25.5c-3.6-10.5-15-16.5-25.5-12.9l-23.7,7.8L232,90.1C229.3,82,222.4,76.9,214.6,76.3z M214.9,166.3
             l16.2,47.7l-47.7,16.2l-16.2-47.7L214.9,166.3z"/>
         </svg>
-      
+
       <span class="footer-icons-text">Join the Layer5 Slack</span></a>
     </div>
-	
+
 	<div class="footer-icons">
       <a class="social__link social__link--calender" href="https://bit.ly/2SbrRhe" >
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="-200 -60 820 740" width="50px" height="50px">
           <path fill="currentColor" stroke="currentColor" stroke-width="3" stroke-miterlimit="10" d="M336 292v24c0 6.6-5.4 12-12 12h-76v76c0
-		  6.6-5.4 12-12 12h-24c-6.6 0-12-5.4-12-12v-76h-76c-6.6 0-12-5.4-12-12v-24c0-6.6 5.4-12 12-12h76v-76c0-6.6 5.4-12 12-12h24c6.6 0 12 
+		  6.6-5.4 12-12 12h-24c-6.6 0-12-5.4-12-12v-76h-76c-6.6 0-12-5.4-12-12v-24c0-6.6 5.4-12 12-12h76v-76c0-6.6 5.4-12 12-12h24c6.6 0 12
 		  5.4 12 12v76h76c6.6 0 12 5.4 12 12zm112-180v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V112c0-26.5 21.5-48 48-48h48V12c0-6.6
 		  5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h128V12c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h48c26.5 0 48 21.5 48 48zm-48 346V160H48v298c0
 		  3.3 2.7 6 6 6h340c3.3 0 6-2.7 6-6z"/>
         </svg>
 
-        <span class="tooltiptext">Subscribe to the community calender</span></a>
```

Comment:
Thanks for this PR, @MoeRayo. 

If we are creating a new CSS class, are we removing the old class?

Reply:
The other icons had the class of `footer-icons-text` except that icon and I figured that must be where the bug was coming from, so I changed the class to match the other icons present.

Reply:
Oh, I see. Sounds real good.